### PR TITLE
added another choose example

### PR DESCRIPTION
**Pull request by Nihvel**
more choose can be used together for creating extended conditions which do not depend from each others

## Proposed change
I added an example that shows how more `choose` can be used within the same automation or script.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Nihvel, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16972).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/scripts.markdown [hunk 1]
```diff
@@ -610,6 +610,63 @@ automation:
 
 {% endraw %}
 
+
+More `choose` can be used together. This is the case of an IF-IF.
+
+{% raw %}
+
+```yaml
+automation:
```
Comment (frenck) [inline source/_docs/scripts.markdown:624]: Please adjust the example to comply with our YAML style guide.

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
I think the example needs a bit more explanation or comments. Due to its size, it is hard to follow for someone new to the syntax.

File: source/_docs/scripts.markdown (hunk 1 above)
Comment (Nihvel) [reply source/_docs/scripts.markdown:624]: Done it, must be correct now

File: source/_docs/scripts.markdown [hunk 2]
```diff
@@ -610,6 +610,62 @@ automation:
 
 {% endraw %}
 
+
+More `choose` can be used together. This is the case of an IF-IF. 
+The following example shows how a single automation can control entities that aren't related to each other but have in common the same trigger.
+When the sun goes below the horizon, the `porch` and `garden` lights must turn on. If someone is watching the TV in the living room, there is a high chance that someone is in that room, therefore the living room lights have to turn on too. Same concept applies for the `studio` room.
```
Comment (frenck) [inline source/_docs/scripts.markdown]:
```suggestion
More `choose` can be used together. This is the case of an IF-IF.

The following example shows how a single automation can control entities that aren't related to each other but have in common the same trigger.

When the sun goes below the horizon, the `porch` and `garden` lights must turn on. If someone is watching the TV in the living room, there is a high chance that someone is in that room, therefore the living room lights have to turn on too. The same concept applies to the `studio` room.
```

File: source/_docs/scripts.markdown [hunk 3]
```diff
@@ -610,6 +610,62 @@ automation:
...
+                    brightness: 255
+                    color_temp: 366
+                  target:
+                    entity_id: light.livingroom
+         # IF another entity not related to the previous, is ON
+        - choose:
+            - conditions:
+                - condition: state
+                  entity_id: binary_sensor.studio_pc
+                  state: on
```
Comment (frenck) [inline source/_docs/scripts.markdown]:
```suggestion
                  state: "on"
```

File: source/_docs/scripts.markdown [hunk 4]
```diff
@@ -610,6 +610,62 @@ automation:
...
+          target:
+            entity_id:
+              - light.porch
+              - light.garden
+        # IF a entity is ON
+        - choose:
+            - conditions:
+                - condition: state
+                  entity_id: binary_sensor.livingroom_tv
+                  state: on
```
Comment (frenck) [inline source/_docs/scripts.markdown]:
```suggestion
                  state: "on"
```

File: source/_docs/scripts.markdown [hunk 5]
```diff
@@ -610,6 +610,62 @@ automation:
...
+
+```yaml
+# Example with "if" and "if"
+automation:
+  - alias: "Turn lights on when the sun gets dim and if some room is occupied"
+      trigger:
+        - platform: numeric_state
+          entity_id: sun.sun
+          value_template: "{{ state.attributes.elevation }}"
+          below: 4
```
Comment (frenck) [inline source/_docs/scripts.markdown]:
```suggestion
        - platform: numeric_state
          entity_id: sun.sun
          attribute: elevation
          below: 4
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @Nihvel 👍
